server/routes/auth.js

```diff
@@ -7,40 +7,31 @@ router.route('/')
   .get(middleware.auth.verify, (req, res) => {
     const preloadedState = {};  
     preloadedState.user = req.user;
-    models.Post.getPostsByUserId(req.user.id)
-      .then(posts => {
-        preloadedState.posts = posts;
+
+    Promise.all([models.Post.getAllPosts(), models.Post.getPostsByUserId(req.user.id)])  
+      .then((results) => {
+        preloadedState.posts = results[0];
+        preloadedState.userPosts = results[1];
+      })
+      .then(() => { 
         res.render('index', {preloadedState});
+      })
+      .catch((err) => {
+
```

Comment:
What's this catch block doing?

Reply:
Corrected with commit. https://github.com/mustachioed-meerkats/mustachioed-meerkats/pull/83/commits/7324033b6eaa963a82488d0bbacb61765db36cda